components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -440,36 +484,38 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
     // Step 4: In-cluster resources set up
     let status = '';
     let getAttempts = 0;
-    const getTimeout = 15000;
+    const getTimeout = 20000;
     do {
       getAttempts++;
+      await wait(getTimeout);
       const curStatus = await Gapi.deploymentmanager.get(this.state.project, deploymentName)
             .catch(err => {
-              this._appendLine('Cluster endpoint not available yet.');
+              this._appendLine('Deployment status not available yet.');
             });
       if (!curStatus) {
-        await wait(getTimeout);
         continue;
       }
       status = curStatus.operation!.status!;
-    } while (status !== 'DONE' && getAttempts < 20);
+      this._appendLine(`Status of ${deploymentName}: ` + status);
+    } while (status !== 'DONE' && getAttempts < 30);
```

Comment:
We need to check other values of the deployment status, and exit the loop with an error if the deployment failed for example.